Sources/StreamChatUI/CommonViews/SendButton/SendButton.swift

```diff
@@ -28,4 +24,13 @@ open class SendButton: _Button, AppearanceProvider {
         let disabledStateImage = appearance.images.sendArrow.tinted(with: buttonColor)
         setImage(disabledStateImage, for: .disabled)
     }
+
+    /// The animation when the `isEnabled` state changes.
+    open func isEnabledChangeAnimation() {
```

Comment:
Would it make more sense for this public function to pass in the `isEnabled` param? Approving anyway more of a small suggestion :)

Reply:
I see.. yes it kinda makes sense 👍